hometask3.py

```diff
@@ -0,0 +1,34 @@
+"""Hometask format and f-strings."""
+import logging
+
+logging.basicConfig(level=logging.INFO, format='[%(asctime)s] %(levelname)s: %(message)s')
```

Comment:
logger was not yet examined by us - but ok if you want  - better to use it in such way
```
_log = logging.getLogger('Main')

console_handler = logging.StreamHandler()
log_format = logging.Formatter('[%(asctime)s %(levelname)s]: %(message)s')
console_handler.setFormatter(log_format)
_log.addHandler(console_handler)
_log.setLevel(logging.DEBUG)

text = 'text'
_log.info(text)
```